flow-defs/utils/media-queries.js.flow

```diff
@@ -15,3 +15,8 @@ declare export var createMediaQueries: (x: {
   desktopOrBigger: string,
   supportsHover: string,
 };
+/**
+ * Scopes the styles to touchable devices.
+ * See: https://stackoverflow.com/questions/12469875/how-to-code-css-media-queries-targeting-all-mobile-devices-and-tablets/42835826#42835826
+ */
+declare export var TOUCHABLE_ONLY: "@media (pointer: coarse), @media (hover: none)";
```

Comment:
For ergonomics, I'd move this to the theme context, as que do with other mq

Reply:
I like that, I missed it.
Indeed that exposes it to webapp and other consumers without duplicating the query